strawberryfields/gbs/similarity.py

```diff
@@ -285,20 +257,16 @@ def event_cardinality(photon_number: int, max_count_per_mode: int, modes: int) -
     return cardinality
 
 
-def p_orbit_mc(orbit: list, graph: nx.Graph, n_mean: float = 5, samples: int = 1000) -> float:

```

Comment:
Also need to update the ordering in the `Args:` section of the docstring.

Reply:
Yes! Will take care of that